spec/integ/crypto.spec.ts

```diff
@@ -1890,4 +1890,70 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
             expect(event.getContent().body).not.toContain("withheld");
         });
     });
+
+    describe("key upload request", () => {
+        beforeEach(() => {
+            jest.useFakeTimers();
+        });
+
+        afterEach(() => {
+            jest.useRealTimers();
+        });
+
+        function listenToUpload(): Promise<number> {
+            return new Promise((resolve) => {
+                const listener = (url: string, options: RequestInit) => {
+                    const content = JSON.parse(options.body as string);
+                    const keysCount = Object.keys(content?.one_time_keys || {}).length;
+                    if (keysCount) resolve(keysCount);
+                    return {
+                        one_time_key_counts: {
+                            signed_curve25519: keysCount ? 60 : keysCount,
+                        },
+                    };
```

Comment:
what's going on here? how did we pick these numbers? why does it depend on `keysCount`? add some comments please!